libs/logic-apps-shared/src/designer-client-services/lib/base/template.ts

```diff
@@ -0,0 +1,12 @@
+import type { IHttpClient } from '../httpClient';
+
+export interface TemplateServiceOptions {
+  baseUrl: string;
+  appId?: string;
+  httpClient: IHttpClient;
+  apiVersions: {
+    subscription: string;
+    gateway: string;
+  };
+  openBladeAfterCreate: () => void;
```

Comment:
This should be present where we have defined the ITemplate service, if this service options is only for standard then defined in the standard template service file.

Reply:
We plan to have the service for consumption as well for v2 so I put that into consideration and left it under base so consumption can import it as well. If we still think this is an issue, I will remove it in the next PR and move it under standard.